src/fides/api/main.py

```diff
@@ -380,6 +192,31 @@ def read_other_paths(request: Request) -> Response:
     return get_admin_index_as_response()
 
 
+@app.on_event("startup")
+async def setup_server() -> None:
+    "Run all of the required setup steps for the webserver."
+
+    log_startup()
+    await run_database_startup()
+    check_redis()
+    initiate_scheduled_batch_email_send()
+
+    logger.debug("Sending startup analytics events...")
+    await send_analytics_event(
+        AnalyticsEvent(
+            docker=in_docker_container(),
+            event=Event.server_start.value,
+            event_created_at=datetime.now(tz=timezone.utc),
+        )
+    )
+
+    if not scheduler.running:
+        scheduler.start()
```

Comment:
I think this still needs to run in the same order as before, at least before the `initiate_scheduled_batch_email_send()`

Reply:
this should be documented then, at least in code comments 🙂 

Would you mind adding code comments detailing what needs to run in what order/what depends on what? Or if you describe it here I can translate it to code comments! If it was easy for me to trip over this not realizing there is a specific order here, we should write it down for future reference

Reply:
I wouldn't recommend reordering anything casually  🙂   

Almost everything we write has a general intentional sequence. Here, I assume the scheduler needs to be started before a task (the email batch send) can be added to it.  I haven't verified that this breaks, but you should.

 Feel free to add code comments where you'd like, but in my opinion, order matters here no more than anywhere else.

Reply:
everything ran for me just fine, but i guess because I kept the order despite rearranging? (tests all passed too) so I think I'll do some further clean up and document clearly that startup order matters and then ship it, if that sounds good to you?

Reply:
It doesn't look like you "kept the order" - unless you have unpushed changes?  The `initiate_scheduled_batch_email_send` is happening before the scheduler is started here. 

 Even if it doesn't error, my unverified concern would be that that batch email send task wasn't actually registered.  The email send is a task that gets scheduled but then only fires once a week -  I don't know if we have a unit test that would specifically break if the scheduling was broken. 

```python
@app.on_event("startup")
async def setup_server() -> None:
    "Run all of the required setup steps for the webserver."

    log_startup()
    await run_database_startup()
    check_redis()
    initiate_scheduled_batch_email_send() ***

    logger.debug("Sending startup analytics events...")
    await send_analytics_event(
        AnalyticsEvent(
            docker=in_docker_container(),
            event=Event.server_start.value,
            event_created_at=datetime.now(tz=timezone.utc),
        )
    )

    if not scheduler.running: ***
        scheduler.start()
```

Reply:
Ah, I see your point. I will update properly to match the current order in `main`